src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/RoboticsDataMapper.java

```diff
@@ -260,7 +260,7 @@ private Solicitor buildApplicantSolicitor(CaseData caseData, String id) {
             }
             respondentParties.add(buildLitigiousParty(
                 caseData.getRespondent2(),
-                caseData.getRespondent1LitigationFriend(),
+                caseData.getRespondent2LitigationFriend(),
```

Comment:
This was adding respondent 1 litigant friend for respondent 2 in the RPA Feed - when respondent 2 hadnt added one.
Updated to correct field `respondent2LitigationFriend`